### DISCUSSION TITLE
Download results to Google Drive source sheet

### DISCUSSION THREAD

**Discussion by jferree-entravision** [jetstreamapp/jetstream#150] [Ideas]
Having the ability to download the results to a new worksheet in the source drive sheet would be amazing!

**Comment by paustint** [comment]
@jferree-entravision - Just to be clear, you are asking about adding a the results to a new worksheet in an existing file - right?

**Reply by jferree-entravision** [reply]
Exactly!

**Reply by paustint** [reply]
I did a bit of digging and it looks like this requires the sheets API and is considered a "sensitive" scope with Google, so they will need to conduct a security review of how the API is being used before it would become approved and available for use without a big scary warning on the Google authorization modal.

I started the process and started looking at what it would take from a code perspective.

**Comment by jferree-entravision** [comment]
Awesome, and thanks for all your work.  Jetstream is an amazing tool!


*Josh Ferree*
Business Applications Manager

5426 North Mesa St.
El Paso, TX 79912
*O* (915) 231-2429   *M* (915) 252-4826
***@***.***

<https://www.linkedin.com/company/entravisionusmedia/>  Connect With Us
<https://www.linkedin.com/company/entravisionusmedia/>
<https://www.linkedin.com/company/entravisionparent/>  Follow Our Global
Page <https://www.linkedin.com/company/entravisionparent/> | Entravision.com
<https://entravision.com>


This message contains information from Entravision Communications
Corporation that may be confidential or privileged. If you are not the
intended recipient, be aware that any disclosure, copying, distribution or
use of the contents of this information is prohibited. If you have received
this transmission in error, please notify me immediately by telephone or
electronic mail. Thank you.
